Sprint 6 Challenge/Sprint 6 Challenge/View Controller/ViewController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ViewController.swift
+//  Sprint 6 Challenge
+//
+//  Created by Spencer Curtis on 8/29/18.
+//  Copyright © 2018 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var lockImageView: UIImageView!
+    @IBOutlet weak var customControl: CustomControl!
+    @IBOutlet weak var resetButton: UIBarButtonItem!
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        resetButton.tintColor = .clear
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        navigationController?.navigationBar.barStyle = .black
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        
+    }
+
+
+    @IBAction func updateLock(_ slider: CustomControl) {
+        
+        if !slider.shouldUnlock {
+            
+        } else {
+            lockImageView.image = #imageLiteral(resourceName: "Unlocked")
+            
+            UIView.animate(withDuration: 0.5) {
+                self.resetButton.tintColor = Appearance.textColor
+            }
+        }
+    }
+    
+    @IBAction func resetButtonTapped(_ sender: Any) {
+        
```

Comment:
Here you will need to add a way to unlock the control, so that the `resetButton` can move everything back to initial/unlocked state. This is as simple as flipping the boolean that you set in `updateLock()`. So, on line 51, we would put `customControl.isEnabled = true`